other-docs/guides/migrating-from-wordpress.md

```diff
@@ -0,0 +1,150 @@
+# Migrating a WordPress Codebase
+
+This guide covers how to migrate a typical WordPress project to HM Platform.
```

Comment:
Should this be "typical legacy HM Platform project" or something like that? The guide is specific to our projects.

Reply:
Yeah the idea was to try keep it _just_ generic enough to call it a migrating WordPress guide that is potentially for all WP projects.